src/Backend/Core/Engine/DataGrid.php

```diff
@@ -117,7 +117,7 @@ public function addColumn(
 
         // known actions that should have a button
         if (in_array(
-            $name,
+            strtolower($name),
```

Comment:
well spotted, sir!

Ive also save the lowercased version for further use in the code